<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor de archivos de texto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Crear y guardar archivo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Abrir y leer archivo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEditor_de_archivos_de_texto">
    <property name="title">
     <string>Editor de archivos de texto</string>
    </property>
   </widget>
   <addaction name="menuEditor_de_archivos_de_texto"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
